src/coldcard.ts

```diff
@@ -269,31 +267,30 @@ class ColdcardMultisigSettingsFileParser extends ColdcardInteraction {
    * 2. derive deeper if necessary (and able) using functionality
    *    from unchained-bitcoin
    *
-   * @param {Object} result - parsed data from ColdcardJSON
-   * @returns {Object} the desired xpub (if possible)
-   *
    */
-  deriveDeeperXpubIfNecessary(result) {
+  deriveDeeperXpubIfNecessary(result: any) {
```

Comment:
I think this could be typed at least as a record as well

```suggestion
  deriveDeeperXpubIfNecessary(result: Record<string, unknown>) {
```

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff